build-phar.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+if [ ! -f box.phar ]; then
+    wget https://github.com/humbug/box/releases/download/3.0.0-beta.4/box.phar -O box.phar
+fi
+
+# lock PHP to minimum allowed version
+composer config platform.php 7.2.0
+composer install --no-dev -q
```

Comment:
this shouldn't be needed, but if you configured the platform just before you should do an `update` no? Also `--prefer-dist` could speed things up a bit as well eventually

Reply:
Hmm, yes, dunno why I changed it.
`--prefer-dist` is implicit for non-dev packages.